misk-feature-testing/src/main/kotlin/misk/feature/testing/FakeFeatureFlagsModule.kt

```diff
@@ -1,34 +1,48 @@
 package misk.feature.testing
 
+import com.google.inject.Provider
 import com.squareup.moshi.Moshi
 import misk.ServiceModule
 import misk.feature.DynamicConfig
 import misk.feature.FeatureFlags
 import misk.feature.FeatureService
+import misk.feature.testing.FakeFeatureFlagsOverrideModule.FakeFeatureFlagsOverride
 import misk.inject.KAbstractModule
+import misk.inject.asSingleton
 import misk.inject.toKey
+import javax.inject.Inject
 import kotlin.reflect.KClass
 
 /**
  * Binds a [FakeFeatureFlags] that allows tests to override values.
+ *
+ * To define overrides (especially test defaults) use [FakeFeatureFlagsOverrideModule].
+ * In a given misk service's test setup, there is one [FakeFeatureFlagsModule] installed and many
+ * [FakeFeatureFlagsOverrideModule] installed.
```

Comment:
Wanna make it so there’s no asymmetry? Ideally I just install `FakeFeatureFlagsModule` one or more times.

Reply:
yeah let me try that!

Reply:
I got stuck since `FakeFeatureFlagsModule` takes an annotation.. it seems to me that there is no way of knowing if we have previously bound a FakeFeatureFlags with the given annotation

Reply:
I was imagining installing an object module for all the `FakeFeatureFlags` bindings but that doesn't work here